src/features/island/bumpkin/components/BumpkinSkillsModal.tsx

```diff
@@ -0,0 +1,242 @@
+                          Cancel
+                        </Button>
+                        <Button onClick={handleClaim} className="text-xs mt-2">
+                          Claim
+                        </Button>
+                      </div>
+                    </div>
+                  ) : (
+                    <>
+                      <div className="flex mb-1 items-center">
+                        <span className="text-shadow text-center text-sm sm:text-base mr-1">
+                          {selectedSkill}
+                        </span>
+                        <img
+                          src={BUMPKIN_SKILL_TREE[selectedSkill].image}
+                          className="object-scale-down"
+                        />
+                      </div>
+
+                      <span className="text-shadow text-center mt-1 text-xxs sm:text-xs mb-1">
+                        {BUMPKIN_SKILL_TREE[selectedSkill].boosts}
+                      </span>
+
+                      {!hasSelectedSkill && (
+                        <>
+                          <div className="border-t border-white w-full pt-1 text-center">
+                            <div
+                              className={classNames(
+                                "flex justify-center flex-wrap items-end mt-2",
+                                {
+                                  "text-error": missingPointRequirement,
+                                }
+                              )}
+                            >
+                              <span className="text-shadow text-center text-xxs sm:text-xs">
+                                Required Skill Points:
+                              </span>
+                              <span className="text-xxs sm:text-xs text-shadow text-center">
+                                {`${availableSkillPoints}/${pointsRequired}`}
+                              </span>
+                            </div>
+                            {skillRequired && (
+                              <div
+                                className={classNames(
+                                  "flex justify-center flex-wrap items-center mt-2",
+                                  {
+                                    "text-error": missingSkillRequirement,
+                                  }
+                                )}
+                              >
+                                <span className="text-shadow text-center text-xxs sm:text-xs">
+                                  Required Skills:
+                                </span>
+                                <img src={requiredSkillImage} />
+                              </div>
+                            )}
+                          </div>
+                          <Button
+                            onClick={() => setShowConfirmButton(true)}
+                            disabled={
+                              missingPointRequirement || missingSkillRequirement
```

Comment:
<img width="374" alt="Screen Shot 2022-09-29 at 2 04 13 pm" src="https://user-images.githubusercontent.com/11745561/192936189-3ecd959d-6d17-4674-b38a-d7c2fec2bbca.png">

Should this be disabled?

Reply:
You seem to be in a strange state here. This is not the path for Trees????

Reply:
This is an old version of the branch 😅